index.bs

```diff
@@ -5623,7 +5624,8 @@ The <dfn export for=commands>emulation.setGeolocationOverride</dfn> command modi
       )
 
       emulation.SetGeolocationOverrideParameters = {
-        coordinates: emulation.GeolocationCoordinates / null,
+        ? coordinates: emulation.GeolocationCoordinates / null,
+        ? error: emulation.GeolocationPositionError,
```

Comment:
To make it mutual exclusive and to enforce that at least one entry needs to be present we probably should use a group choice:

```suggestion
    (
      (coordinates: emulation.GeolocationCoordinates / null) //
      (error: emulation.GeolocationPositionError)
    ),
```

It could also be moved out to a separate definition for a better readability.